gsy_e_sdk/utils.py

```diff
@@ -337,3 +337,27 @@ def get_name_from_area_name_uuid_mapping(area_name_uuid_mapping, asset_uuid):
             if area_uuid == asset_uuid:
                 return area_name
     return None
+
+
+def get_assets_name(registry: Dict) -> Dict:
+    """
+    Parse the grid tree and return all registered assets / markets
+    Wrapper for _get_assets_name
+    """
+    if registry == {}:
+        return {}
```

Comment:
The registry returned by `get_configuration_registry()` can also be `None` in case of errors. I'd suggest replacing this with
```suggestion
    if not registry:
        return {}
```